docs/calico.md

```diff
@@ -219,6 +219,15 @@ calico_vxlan_mode: 'Never'
 
 If you use VXLAN mode, BGP networking is not required. You can disable BGP to reduce the moving parts in your cluster by `calico_network_backend: vxlan`
 
+## Configuring interface MTU
+
+This is an advanced topic and should usually be left to defaults as Calico is smart enough to deal with the defaults. If you do need to set up a custom MTU you can change `calico_veth_mtu` as follows:
+
+* If Wireguard is enabled, set to your physical network MTU - 60 (i.e. 1440 or 8940)
```

Comment:
I cannot follow what means `60` here.
The default value of calico_veth_mtu is 1440, and such values seem good for me for MTU.
but I cannot get the point of small values like `60` for that.
Could you explain that?

Reply:
I think this needs a reformulation as it does not mean to set the MTU to 60 but subtract 60 from the physical network MTU when computing the value for `calico_veth_mtu`. Actually this was the confusion on https://github.com/kubernetes-sigs/kubespray/issues/7747